src/GoogleAuthenticator/Helper.php

```diff
@@ -30,13 +32,31 @@ class Helper
     protected $authenticator;
 
     /**
-     * @param string                  $server
-     * @param BaseGoogleAuthenticator $authenticator
+     * @var array
      */
-    public function __construct($server, BaseGoogleAuthenticator $authenticator)
+    protected $forcedForRoles;
+
+    /**
+     * @var array
+     */
+    protected $ipWhiteList;
+
+    /**
+     * @var AuthorizationCheckerInterface
+     */
+    private $authorizationChecker;
+
+    /**
+     * @param array $forcedForRole Roles that need to force use 2FA authorization
+     * @param array $ipWhiteList   IP's that will skip 2FA authorization
+     */
+    public function __construct($server, array $forcedForRoles = [], array $ipWhiteList = [], BaseGoogleAuthenticator $authenticator, AuthorizationCheckerInterface $authorizationChecker)
```

Comment:
Please linebreak this

Reply:
You can't add optional arguments in the middle of the signature, move them to the end please